src/gm-apclientpp.cpp

```diff
@@ -684,3 +719,192 @@ double apclient_death_link(const char* cause)
         return GM_FALSE;
     return GM_BOOL(apclient && apclient->Bounce(data_j, {}, {}, { "DeathLink" }));
 }
+
+inline int key_to_int(const char* key) {
+    int i = 0;
+    while (*key) {
+        i *= 10;
+        i += *key++ - '0';
+    }
+    return i;
+}
+
+double apclient_json_proxy(const double proxy, const char* key)
+{
+    const std::lock_guard<std::mutex> lock(mut);
+    unsigned int int_proxy = (int)proxy;
```

Comment:
Similar to the json key thing.
```suggestion
    size_t int_proxy = (size_t)proxy;
```
The same snippet exists a couple more times below.